index.html

```diff
@@ -185,8 +221,8 @@ <h2>CONTACT US</h2>
             <li><a href="https://twitter.com/GraceAbike1"><img src="imgs/Icon-Twitter.png" alt="Twitter"></a></li>
             <li><a href="#"><img src="imgs/Icon-Medium.png" alt="Medium"></a></li>
           </ul>
-        <a href="#" class="email">mara234216@gmail.com</a>
-      </div>
+          <a href="#" class="email">mara234216@gmail.com</a>
```

Comment:
kindly modify the email at the footer because it seems to overlap with the contact form.

Your design:
![image](https://user-images.githubusercontent.com/20554583/186684084-bc4e0eb1-a386-40ad-bd2d-973dbf257cd3.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/186684218-273ac796-fd59-4dbe-b347-fa38509f4e86.png)

Reply:
@emmiiorji 
<img width="946" alt="image" src="https://user-images.githubusercontent.com/88346506/186688228-d1ea6077-23e1-4cb3-a754-0480b248c6a5.png">

Reply:
Hi @GraceAbike,

Kindly take a look at this illustration:

https://user-images.githubusercontent.com/20554583/186714281-1d454d62-6f76-4b5e-b2fe-d73b366c0ba9.mp4

A closer look reveals more about how the responsiveness on tablet size screens was handled.

Reply:
@emmiiorji okay thank you